content/posts/pyodide-joins-open-source-collective.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Pyodide becomes member of the Open Source Collective"
+date: 2021-11-10T13:13:04+01:00
+draft: true
+tags: ["announcement"]
+author: "Roman Yurchak on behalf of the Pyodide team"
```

Comment:
We need to decide how we formulate this in project announcements blog posts. We could just put "The Pyodide team" as the author, but I feel the person who writes the post should still appear somewhere. Also in practice most of these would likely be co-written / changed through review, so here it would likely be Hood and me?